pkgs/by-name/no/nominatim-ui/package.nix

```diff
@@ -2,8 +2,30 @@
   lib,
   stdenv,
   fetchFromGitHub,
+  fetchYarnDeps,
+  writeText,
+
+  fixup-yarn-lock,
+  nodejs,
+  yarn,
+
+  # Custom application configuration placed to theme/config.theme.js file
+  # For the list of available configuration options see
+  # https://github.com/osm-search/nominatim-ui/blob/master/dist/config.defaults.js
+  customConfig ? null,
 }:
 
+let
+  configFile =
+    if customConfig != null then
+      writeText "config.theme.js" customConfig
+    else
+      writeText "config.theme.js" ''
+        // Default configuration
+        Nominatim_Config.Nominatim_API_Endpoint='https://localhost/';
```

Comment:
Would it be better to make this a HTTP URL, to avoid dealing with certificates, and link to 127.0.0.1 instead of localhost to avoid having to deal with name resolution?

Reply:
SSL certificates are not a problem with NixOS's Nginx module. Self-signed certificate is automatically generated as a fallback if Let's Encrypt fails. Replacing localhost with `127.0.0.1` makes a sense.